src/neuroconv/datainterfaces/behavior/deeplabcut/_dlc_utils.py

```diff
@@ -334,122 +373,60 @@ def _write_pes_to_nwbfile(
         )
         pose_estimation_series.append(pes)
 
-    camera_name = pose_estimation_name
-    if camera_name not in nwbfile.devices:
+    # Extract device information
+    device_name = container_metadata.get("devices", [f"Camera{container_name}"])[0]
+    device_metadata = pose_estimation_metadata.get("Devices", {}).get(device_name, {})
+
+    # Create or get the device
+    if device_name not in nwbfile.devices:
         camera = nwbfile.create_device(
-            name=camera_name,
-            description="Camera used for behavioral recording and pose estimation.",
+            name=device_name,
```

Comment:
what about the other device kwargs (ex. manufacturer)?

Reply:
I don't want to move too much in this direction because I really prefer to specify those as a device model instead of as a device:

https://github.com/NeurodataWithoutBorders/nwb-schema/issues/607

Reply:
I worry that DeviceModel won't be merged for a while yet. And it should be a really simple fix to just grab the kwargs from the metadata.

Reply:
Let's wait for the next LBNWL or group meeting to settle this then.

Reply:
I think maybe you're misunderstanding my request? Let me make it more concrete...

Reply:
Instead of
```python
camera = nwbfile.create_device(
    name=device_name,
    description=device_metadata["description"],
)
```

just use 
```python
camera = nwbfile.create_device(**device_metadata)
```

and let the user work out which device metadata fields to include or not.  No need to worry about which ones might get moved to DeviceModel.

Reply:
No, I think I understand. I am thinking on whether we should allow the writing of attributes which will be deprecated.

Reply:
Ah, I see. I still think it shouldn't be a big deal to allow this, but it can wait until we hear from the group.